src/main/resources/META-INF/rewrite/spring-boot-34-properties.yml

```diff
@@ -0,0 +1,127 @@
+      oldPropertyKey: management.endpoint.httpexchanges.enabled
+      newPropertyKey: management.endpoint.httpexchanges.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.info.enabled
+      newPropertyKey: management.endpoint.info.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.integrationgraph.enabled
+      newPropertyKey: management.endpoint.integrationgraph.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.liquibase.enabled
+      newPropertyKey: management.endpoint.liquibase.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.logfile.enabled
+      newPropertyKey: management.endpoint.logfile.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.loggers.enabled
+      newPropertyKey: management.endpoint.loggers.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.mappings.enabled
+      newPropertyKey: management.endpoint.mappings.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.metrics.enabled
+      newPropertyKey: management.endpoint.metrics.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.prometheus.enabled
+      newPropertyKey: management.endpoint.prometheus.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.quartz.enabled
+      newPropertyKey: management.endpoint.quartz.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.sbom.enabled
+      newPropertyKey: management.endpoint.sbom.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.scheduledtasks.enabled
+      newPropertyKey: management.endpoint.scheduledtasks.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.sessions.enabled
+      newPropertyKey: management.endpoint.sessions.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.shutdown.enabled
+      newPropertyKey: management.endpoint.shutdown.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.startup.enabled
+      newPropertyKey: management.endpoint.startup.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoint.threaddump.enabled
+      newPropertyKey: management.endpoint.threaddump.access
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.endpoints.enabled-by-default
+      newPropertyKey: management.endpoints.access.default
```

Comment:
So this would break, because the original has a `true/false` argument, whereas the replacement takes different values. 

You'd want to instead move from `false` to `none` and `true` to `read-only`, but that's not expressible with `ChangeSpringPropertyKey`. 

Instead for now I'd propose to [do no harm](https://docs.openrewrite.org/authoring-recipes/recipe-conventions-and-best-practices#do-no-harm) and remove these incomplete migrations.
```suggestion
```